### PR TITLE
Create Task7.py

### PR DESCRIPTION
**Pull request by Solo-Anna**

### CODE REVIEW

File: Tasks/Salanenka/Task7/Task7.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return sum
+
+d = [1, 2, [2, 4, [[7, 8], 4, 6]]]
+print (summa(d))
+
+def fib(n):
+    def fi (n):
+        if n<=1: 
+            return n
+        return (fi(n-1)+fi(n-2))
```
Comment (EugeneDenkevich) [inline Tasks/Salanenka/Task7/Task7.py:17]: Hi!) It's realy difficult to understand fo me! =) I debuged it for almost 20 times before catch the logic.)) Cool solution!)

File: Tasks/Salanenka/Task7/Task7.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+def summa (a):
+    sum=0
+    for i in a:
+        if type(i) == int:
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Salanenka/Task7/Task7.py:4]: please use the **isinstance** method to check a type, it works faster

File: Tasks/Salanenka/Task7/Task7.py (hunk 1 above)
Comment (MikitaTsiarentsyeu) [inline Tasks/Salanenka/Task7/Task7.py:17]: good, I see, that you understand the recursion

**Review by MikitaTsiarentsyeu — APPROVED** [review]
great job, the score is 100

**Pull request merged**